app/Http/Controllers/Api/V1/Uom/UomController.php

```
@@ -1,57 +1,52 @@
 use App\Http\Controllers\Controller;
 use App\Http\Requests\FilterUomRequest;
 use App\Http\Requests\StoreUomRequest;
 use App\Http\Requests\UpdateUomRequest;
+use App\Http\Resources\UomListAllResource;
+use App\Http\Resources\UomListResource;
 use App\Models\Uom;
-use App\Services\UomService;
-
 class UomController extends Controller
 {
     public function index(FilterUomRequest $request)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)